nio.conf.example

```diff
@@ -0,0 +1,673 @@
+# n.io project configuration
+#
+[user_defined]
+
+# host and port to launch rest web server on
+NIOHOST=0.0.0.0
+NIOPORT=8181
+
+# Path to project root, leave blank to auto detect - it will use the root
+# specified when running the binary (passed with -r or --root)
+PROJECT_ROOT=
+
+INSTANCE_NAME=local
+INSTANCE_ID=local
+
+# Pubkeeper Client
+PK_HOST=your-system-id.pubkeeper.nio.works
+PK_PORT=443
+PK_TOKEN=
+PK_SECURE=True
+PK_CA_CHAIN=
+
+# WebsocketBrew Variables
+WS_HOST=your-system-id.websocket.nio.works
+WS_PORT=443
+WS_SECURE=True
+
+# Pubkeeper Server
+PKS_JWT_SECRET=
+PKS_ADDR=
+PKS_INTERFACE=
+PKS_PORT=
+
+# ZMQ / ZMQBrew Variables
+# your IP address that other nodes can reach you on
+IP=127.0.0.1
+
+# ZMQ Encryption settings
+# optional comma separated list of allowed ip addresses, if none specified,
+# this extra layer of security is not enforced.
+# This setting will only take effect if the key encryption is enabled with
+# the settings below.
+ALLOWED_IP_ADDRESSES=
+
+# used for zmq strongest security model
+# This is a setting that must be set on an instance containing publishers.
+# Every subscriber's public key should go in this directory.
+# The public keys for each subscriber should be paired with the subscriber's
+# secret that is set on that host's CLIENT_SECRET_CERT
+PUBLISHER_PUBLIC_KEYS_DIR=etc/public_keys
+
+# publisher server-like certificates
+# The publisher instance must set it's server secret key.
+# All subscriber nodes only need to set the server's public key.
+PUBLISHER_SERVER_PUBLIC_CERT=etc/public_keys/server.key
+PUBLISHER_SERVER_SECRET_CERT=etc/private_keys/server.key_secret
+
+# subscriber client-like certificates
+# This client's secret key. This will be paired with the public key
+# that should be copied to the broker instance's public keys folder
+SUBSCRIBER_CLIENT_SECRET_CERT=etc/private_keys/client.key_secret
+
+# ZMQ based communication module - Legacy
+
+# the ip address of the master broker
+COMHOST=127.0.0.1
+# Are you the broker?
+MASTER_BROKER=True
+# ports used by master broker to communicate with its clients
+XPUB_PORT=9000
+XSUB_PORT=9001
+
+
+[environment]
+
+# Location of custom extensions for auto-discovery
+#
+#blocks=blocks
```

Comment:
Should all of these default values be un-commented in the example?

Reply:
Hmm that's an interesting thought. I'll check around some other places and see what best practices look like

Reply:
What I've seen before is default values are shown in the file but commented out. Like how we have it.